jersey-sample-poc/src/main/java/com/mh/jerseysamplepoc/servlet/PostsServlet.java

```diff
@@ -0,0 +1,42 @@
+package com.mh.jerseysamplepoc.servlet;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.mh.jerseysamplepoc.RestClient;
+import com.mh.jerseysamplepoc.vo.Posts;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.ArrayList;
+
+@WebServlet(urlPatterns = {"/PostsServlet"}, name = "PostsServlet")

```

Comment:
follow lower camelCase syntax. /postsServlet

Reply:
Changed url pattern to /postsServlet and name to postsServlet respectively.